abyss_filters/lvars_alias.py

```diff
@@ -50,6 +56,32 @@ def set_unique_name(var_x, new_name, lvars):
     var_x.set_user_name()
     return new_name
 
+def get_expr_name(expr):
+    name = expr.print1(None)
+    name = ida_lines.tag_remove(name)
+    name = ida_pro.str2user(name)
+    return name
+
+def get_expr_name_and_member(expr):
+    name = get_expr_name(expr)
+    if "->" in name:
+        return name.split("->")
+    elif "." in name:
+        return name.split(".")
+    return None
+
+# XXX - There should be a better way than parsing the text to get the type

```

Comment:
In reference to your comment in line 73 and after our earlier private discussions where we've agreed on the error-proneness of this approach, I have got a hard time accepting this pull request. Don't get me wrong, I appreciate the effort and yet, I would like to avoid having to rely on parsing textual representation. Feel free to implement a separate filter based on this approach however, the "func_colorizer.py" filter might serve as a good example!
I do agree the code in your PR will work in some (if not most) of the cases, but it would still fail if multiple structure members are accessed for example (struct->member.member etc.).
I think retrieving the structure type and parsing its members using SDK functions is the correct way of implementing this new functionality.